apps/src/templates/studioHomepages/SectionsTable.jsx

```diff
@@ -206,7 +207,9 @@ const SectionsTable = React.createClass({
                 </td>
               )}
               <td style={{...styles.col, ...(isRtl? styles.sectionCodeColRtl: styles.sectionCodeCol)}}>
-                {section.code}
+                <nobr>{section.login_type == SectionLoginType.clever ? i18n.loginTypeClever() :
+                    section.login_type == SectionLoginType.google_classroom ? i18n.loginTypeGoogleClassroom() :
+                        section.code}</nobr>
```

Comment:
I'd prefer we used the `white-space: nowrap` CSS property here to the `<nobr>` tag, [which is apparently nonstandard](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nobr).